salt/states/junos.py

```diff
@@ -161,7 +161,7 @@ def commit(name, **kwargs):
 
 
 @resultdecorator
-def rollback(name, id, **kwargs):
+def rollback(name, id=0, **kwargs):
```

Comment:
From internal talks, 'id' is a reserved word in Salt state files.  A ticket has been opened with Documentation to have this clearly stated. I know it is 'id' in the code but perhaps changing 'id' to 'd_id' at this point might be better, similar to 
```
197 @resultdecorator
198 def diff(name, d_id=0, **kwargs):
```
Otherwise the change looks fine

Reply:
how about r_id corresponding to rollback id. changed via https://github.com/saltstack/salt/pull/57410/commits/a498be48686ad1d7ab8908077b45daf0ed1971b0

Reply:
Looks good